src/coreclr/debug/daccess/request.cpp

```diff
@@ -3676,7 +3676,7 @@ ClrDataAccess::GetSyncBlockData(unsigned int SBNumber, struct DacpSyncBlockData
 
         if (ste->m_SyncBlock != NULL)
         {
-            SyncBlock *pBlock = PTR_SyncBlock(ste->m_SyncBlock);
+            PTR_SyncBlock pBlock = ste->m_SyncBlock;
             pSyncBlockData->SyncBlockPointer = HOST_CDADDR(pBlock);
 #ifdef FEATURE_COMINTEROP
             if (pBlock->m_pInteropInfo)
```

Comment:
An odd thing is - did you see a defined error around this? I expect PTR -> * to do a debugger-local copy. As far as I see m_Monitor and m_Link are in the same level of non-DACization: they are things that once copied they "make sense". Not a fan, but I don't see where this one doesn't work.